.github/workflows/CI/revad-compose.yml

```diff
@@ -1,7 +1,7 @@
 version: "3.9"
 services:
   revad:
-    image: cs3org/revad # 'latest' tag is temporarily not available
+    image: cs3org/revad:v2.0.0 # 'latest' tag is temporarily not available
```

Comment:
Note that reva v2.x tags are used by ownCloud for OCIS, and are significantly different (and incompatible) with v1.x tags, in particular from a StorageProvider API perspective. The current "latest" reva tag is 1.24.